lib/trento_web/router.ex

```diff
@@ -88,6 +88,8 @@ defmodule TrentoWeb.Router do
 
       get "/about", AboutController, :info
 
+      get "/activity_log", ActivityLogController, :get_activity_log
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Being we are responding with the list of entries for the log, would it be preferable this path?

```suggestion
      get "/activity_log/entries", ActivityLogController, :get_activity_log
```

Reply:
Can revisit this later, if there is a stronger need/motivation.